### DISCUSSION TITLE
[#980315] A function that allows you to select block types in a hotbar format using the brush tool

### DISCUSSION THREAD

**Discussion by kiku09020** [Mojang/minecraft-editor#47] [Feedback]
It can be a bit difficult to repeatedly change block types. Therefore, I would like to have the ability to select multiple blocks in a hotbar format, just like in regular Minecraft. Additionally, since there is no eraser function, I would like to have the option to always select air blocks as a substitute.

(This is translated English.I'm sorry if my grammar was incorrect)

**Comment by Landy005** [comment]
Or maybe just a history bar for the past, like, 9 blocks used?

**Reply by minecraftchloe** [reply]
Recently used blocks has been spec'd out in an upcoming block picker revamp!
